e2e/tests/home/homepage.spec.ts

```diff
@@ -1,51 +1,142 @@
 
 jest.setTimeout(60 * 1000);
 
-const configs = require('../../resources/config');
+const configs = require('../../resources/workbench-config');
 
-describe.skip('Home page tests:', () => {
+describe('Home', () => {
   let browser;
   let page;
-  let cookies;
 
   beforeAll(async () => {
-    browser = await PuppeteerLaunch();
+    browser = await launchBrowser();
   });
 
   beforeEach(async () => {
     page = await browser.newPage();
     await page.setUserAgent(configs.puppeteerUserAgent);
-    await page.setDefaultNavigationTimeout(60000);
   });
 
   afterEach(async () => {
     await page.close();
+    await page.waitFor(1000);
   });
 
   afterAll(async () => {
     await browser.close();
   });
 
-  test('Homepage is the landing page after sign in', async () => {
+  test('Homepage is the landing page after Sign In', async () => {
+    // Enable networks requests inspection
+    await page.setRequestInterception(true);
+
+    // Following network requests expected to happen
+    const targetRequestsUrls = [
+      configs.apiBaseUrl+'/status-alert',
+      configs.apiBaseUrl+'/config',
+      configs.apiBaseUrl+'/profile',
+      configs.apiBaseUrl+'/workspaces',
+      configs.apiBaseUrl+'/cdrVersions',
+      configs.apiBaseUrl+'/workspaces/user-recent-workspaces',
+      configs.apiBaseUrl+'/workspaces/user-recent-resources',
+    ];
+    const requestsUrls = [];
+    const requestsFailed = [];
+    const pageErrors = [];
+
+    page.on('request', request => {
+      request.continue();
+    });
+
+    // Catch console log errors
+    page.on('pageerror', err => {
+      pageErrors.push(`${err.toString()}`);
+    });
+
+    page.on('response', response => {
+      const request = response.request();
+      const url = request.url().split('?')[0].split('#')[0];
+      const status = response.status();
+      if (targetRequestsUrls.includes(url)) {
+        requestsUrls.push(request.url());
+        // console.log('response url:', url, 'status:', status);
+        if (status !== 200) {
+          requestsFailed.push(request.url());
+        }
+      }
+    });
+
     const loginPage = new GoogleLoginPage(page);
     await loginPage.login();
-    const homePage = new Home(page);
-    await homePage.waitForReady();
+
+    const home = new HomePage(page);
+    await home.waitForReady();
     const title = await page.title();
     expect(title).toMatch('Homepage');
-    cookies = await page.cookies();
+
+    // Check targeted api succeeded
+    expect(requestsUrls).toEqual(expect.arrayContaining(targetRequestsUrls));
+    // Expect zero request fail
+    expect(requestsFailed.length).toEqual(0);
+    // Expect zero page error
+    expect(pageErrors.length).toEqual(0);
   });
 
-  test('Sign in with cookies', async () => {
-    await page.setCookie(...cookies);
-    const workspaces = new WorkspacePage(page);
-    await workspaces.goURL();
+  test('Workspace cards have same UI size', async () => {
```

Comment:
if workspace card were ever not rendered correctly, say size == 0, this test would fail immediately. 
also I was planning to add tests to check access-level, timestamp, names.